tests/sentry/replays/consumers/recording_consumer/test_consumer.py

```diff
@@ -156,6 +164,21 @@ def test_basic_flow_uncompressed(self):
         self.project.refresh_from_db()
         assert self.project.flags.has_replays
 
+        # mock_onboarding_task.assert_called_with(
```

Comment:
are these commented out lines needed?

Reply:
they're meant to be testing things. i need to push i think

Reply:
ah, no those are extra, we've got the assertions in the previous test case.